x-pack/test/api_integration/apis/metrics_ui/infra_log_analysis_validation_log_entry_datasets.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import {
+  validateLogEntryDatasetsRequestPayloadRT,
+  validateLogEntryDatasetsResponsePayloadRT,
+} from '../../../../plugins/infra/common/http_api/log_analysis/validation/datasets';
+import { decodeOrThrow } from '../../../../plugins/infra/common/runtime_types';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function ({ getService }: FtrProviderContext) {
+  const esArchiver = getService('esArchiver');
+  const supertest = getService('supertest');
+
+  describe('API /infra/log_analysis/validation/log_entry_datasets', () => {
+    before(() =>
+      esArchiver.load('x-pack/test/functional/es_archives/infra/8.0.0/logs_and_metrics')
+    );
+    after(() =>
+      esArchiver.unload('x-pack/test/functional/es_archives/infra/8.0.0/logs_and_metrics')
+    );
+
+    it('works', async () => {
+      const response = await supertest
+        .post('/api/infra/log_analysis/validation/log_entry_datasets')
```

Comment:
How about we use the constant from `common` here? It's exported alongside the runtime type used below.

Reply:
Sure thing, will fix!